paddle/fluid/operators/lookup_table_op.h

```diff
@@ -25,56 +28,88 @@ namespace operators {
 using Tensor = framework::Tensor;
 using LoDTensor = framework::LoDTensor;
 using SelectedRows = framework::SelectedRows;
+using DDim = framework::DDim;
+
+static constexpr int64_t kNoPadding = -1;
+
+inline size_t getIndex(const std::vector<int64_t> &rows, int64_t value) {
```

Comment:
`getIndex` is not style-compatible.  Should be `GetIndex`.

Also, GetIndex is a too general name; we might need a specific name, or move it into a class, which represents the context of the name.

Reply:
Ok, will move it to selected_rows.cc